neo/dense.nim

```diff
@@ -1070,6 +1072,152 @@ overload(gehrd, sgehrd, dgehrd)
 overload(orghr, sorghr, dorghr)
 overload(hseqr, shseqr, dhseqr)
 
+overload(gesdd,sgesdd,  dgesdd)
+
+func newSeqUninit*[T](len: Natural): seq[T] {.inline.} =
```

Comment:
Thank you, this seems great to me! The only comment on my side: shouldn't this function be private to the module? I don't see it used elsewhere and is not a function that it makes sense to expose

Reply:
you are correct it doesn't need to be public so I made another commit making it a private function.